taxi/tests/test_models.py

```diff
@@ -0,0 +1,91 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from taxi.models import Manufacturer, Car
+
+User = get_user_model()
+
+
+class ManufacturerModelTest(TestCase):
+    def setUp(self):
+        self.manufacturer = Manufacturer.objects.create(
+            name="Toyota",
+            country="Japan"
+        )
+
+    def test_manufacturer_creation(self):
+        self.assertEqual(self.manufacturer.name, "Toyota")
+        self.assertEqual(self.manufacturer.country, "Japan")
+
+    def test_manufacturer_str(self):
+        self.assertEqual(str(self.manufacturer), "Toyota Japan")
+
+    def test_manufacturer_unique_name(self):
+        with self.assertRaises(Exception):
+            Manufacturer.objects.create(name="Toyota", country="USA")
+
+
+class DriverModelTest(TestCase):
+    def setUp(self):
+        self.driver = User.objects.create_user(
+            username="driver1",
+            first_name="John",
+            last_name="Doe",
+            license_number="ABC123456"
+        )
+
+    def test_driver_creation(self):
+        self.assertEqual(self.driver.username, "driver1")
+        self.assertEqual(self.driver.first_name, "John")
+        self.assertEqual(self.driver.last_name, "Doe")
+        self.assertEqual(self.driver.license_number, "ABC123456")
+
+    def test_driver_str(self):
+        self.assertEqual(str(self.driver), "driver1 (John Doe)")
+
+    def test_driver_unique_license_number(self):
+        with self.assertRaises(Exception):
```

Comment:
Consider specifying the exception type (e.g., `IntegrityError`) to ensure the test accurately captures the intended behavior when testing for unique `license_number`.